CMakeLists.txt

```diff
@@ -168,6 +168,14 @@ include( CMakePushCheckState )
 include( GNUInstallDirs )
 include( TestBigEndian )
 
+cmake_dependent_option(
+   ${_OPT}has_updates_check
+   "Has a builtin support for checking of updates"
```

Comment:
I suggest a comment string consistent with others recently added to this file:
"Build networking features into Audacity"
"Build support for sending errors to Sentry"

That is, "Build..."

Reply:
Renamed on "Build updates checking features into Audacity".